### ISSUE TITLE
Get Account Daily Spent Amount

### ISSUE THREAD

**Issue by divyeshgajja** [googleads/google-ads-php#704]
[DO NOT INCLUDE ANY PERSONAL OR SENSITIVE DATA - MAKE SURE TO REDACT CONTENT WHEN NECESSARY]

**Your question:**
Hello Sir ,
Can you Please Send me Details regarding How to Get my account Daily Spent, using Google Ads API in PHP,
I go through all the documents but I am not finding any solution for this so Please send me details How to Do it.

**Comment by fiboknacky** [comment]
Do you mean you want something like what's described in [this article](https://support.google.com/google-ads/answer/6385083?hl=en)?

**Comment by divyeshgajja** [comment]
Yes but i need daily spent for my account not particular campaign. I need full account average daily spend amount.

**Comment by fiboknacky** [comment]
Do you mean you want a budget you set or real cost that the account spends?

**Comment by divyeshgajja** [comment]
I need account spends.

**Comment by fiboknacky** [comment]
As you mean the cost, you can fetch [`metrics.cost_micros`](https://developers.google.com/google-ads/api/fields/v9/customer?hl=en#metrics.cost_micros) to retrieve the value with the resource `customer`.

Refer to [this file](https://github.com/googleads/google-ads-php/blob/main/examples/Reporting/GetKeywordStats.php) for an example of fetching reports.
See [this section](https://developers.google.com/google-ads/api/docs/reporting/overview) if you need a refresh in general.
